src/AudioIO.h

```diff
@@ -76,12 +80,12 @@ struct AudioIOEvent {
 };
 
 struct TransportTracks {
-   WaveTrackArray playbackTracks;
-   WaveTrackArray captureTracks;
+   WritableSampleTrackArray playbackTracks;
```

Comment:
I wonder why playback tracks should be writable

Reply:
Later commit addressed that, yes?

Reply:
Yes!